### PR TITLE
add profile-namespace variable to bikesharing example

### PR DESCRIPTION
**Pull request by AyushSinha5588**
Provide a clear and concise description of the content changes you're proposing. List all the changes you are making
to the content.

- Added new cell for PROFILE_NAMESPACE variable under PREDICTION section

Closes #...

> If there is no issue related to this PR, kindly create one first to describe the motivation behind these changes.

Checklist:

- [ ] I have checked that my enhancements are not duplicates of existing content changes or additions.
- [ ] I have tested the changes in a working environment to ensure they function as intended.
- [ ] I have followed the [style guide](https://github.com/HPEEzmeral/ezua-tutorials/blob/develop/CONTRIBUTING.md#style-guide)
      outlined in the contribution guidelines.

Reviewer's Tasks (for maintainers reviewing this PR):

- [ ] Verify that the tutorial functions correctly in a live environment.
- [ ] Verify that the updated content aligns with the [style guide](https://github.com/HPEEzmeral/ezua-tutorials/blob/develop/CONTRIBUTING.md#style-guide)
      in the contribution guidelines.
- [ ] Check for consistency, grammar, and clarity throughout the updated content.
- [ ] Check that the related GitHub issue is up-to-date.

### CODE REVIEW

**Comment by AyushSinha5588** [comment]
This PR is raised to fix the following error coming when running the bikesharing example in Mlflow: 

image

The reason for this error is the namespace we are setting in the **kserve inferenceservice**. The namespace value is read from the **USER** env variable which is set to **hpedemo-user01**. But after recent changes w.r.t profile-namespace, this is no longer valid. So after discussing with @CybervisionVolodymyrPogorelov , we have put this temporary change where user will be responsible for setting the profile-namespace while running the example.

**Review by akravacyber — APPROVED** [review]

**Review by jacob-gordon-hpe — APPROVED** [review]

**Review by Sourabh0511 — APPROVED** [review]
LGTM

File: Data-Science/MLflow/bike-sharing-prediction.ipynb [hunk 1]
```diff
@@ -11,6 +11,16 @@
...
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set the profile namespace below\n",
+    "PROFILE_NAMESPACE = \"<profile-namespace>\""
```
Comment (ask664) [inline Data-Science/MLflow/bike-sharing-prediction.ipynb:21]: we should have to get profile_name through kubectl get ns and we should have to automate this.

**Comment by AyushSinha5588** [comment]
Updated the code as per @ask664 suggestion. 

Now reading the namespace from the ConfigMap. 
 
Capture

File: Data-Science/MLflow/bike-sharing-prediction.ipynb (hunk 1 above)
Comment (AyushSinha5588) [reply Data-Science/MLflow/bike-sharing-prediction.ipynb]: @ask664  I have updated the code to read the namespace from the configmap 'user-info-cm'.

**Review by CybervisionVolodymyrPogorelov — APPROVED** [review]

**Pull request merged**